packages/core/src/Stores/notification-store.js

```diff
@@ -55,8 +55,9 @@ export default class NotificationStore extends BaseStore {
             ],
             () => {
                 if (
-                    root_store.client.landing_companies &&
-                    Object.keys(root_store.client.landing_companies).length > 0
+                    !root_store.client.is_logged_in ||
+                    (Object.keys(root_store.client.account_status).length > 0 &&
+                        Object.keys(root_store.client.landing_companies).length > 0)
```

Comment:
Added a check for account_status not to be an empty object to eliminate the console error.
No need to check if account_status or landing_companies are undefined because they are never undefined here (first they are empty objects (not undefined) set in client store, then they are either filled with properties, or not).